open-metadata-implementation/user-interfaces/presentation-server/cra-client/package.json

```diff
@@ -24,13 +24,18 @@
   },
   "devDependencies": {},
   "scripts": {
-    "start": "react-scripts start",
+    "start": "HTTPS=true react-scripts start",
```

Comment:
in the official documentation, here [0], it says that in order to deploy the generated build, production code, you should set up your favourite HTTP server and they offer an example using "serve" or a nodejs with express static folder approach.

having this command [1] as a production command for starting the ui is actually starting a development server with HTTPS flag, adding `--prod` will also minify and uglify the code(as an example), but still, it is wrongly approached.

`react-scripts` should be used for development only and for bundling the code into production, like preparing the ENV variables and assets.

it also tells here [2] that, and i quote,
> This will simply start the basic, ExpressJS server included with the react-server package. It's not a total solution yet! Don't worry, we'll get there later in the guide.
which means there should be at least another layer of nginx as they say in the tutorial later.


[0] - https://create-react-app.dev/docs/deployment
[1] - `HTTPS=true react-scripts start`
[2] - https://react-server.io/docs/guides/production

Reply:
Just pushed an update to remove the HTTPS flag for now. The correct build approach is going to be implemented in the next PR

Reply:
The purpose of this PR is to implement react-scripts to enable more efficient development for the other UI authors before the build is finalized